libraries/core_libs/consensus/include/pbft/round.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "pbft/round_face.hpp"
+#include "pbft/step/certify.hpp"
+#include "pbft/step/filter.hpp"
+#include "pbft/step/finish.hpp"
+#include "pbft/step/polling.hpp"
+#include "pbft/step/propose.hpp"
+#include "pbft/step/step.hpp"
+
+namespace taraxa {
+
+class Round : public RoundFace {
```

Comment:
what is RoundFace base class good for when no other class is derived from it ?

Reply:
This is needed to not create cyclic dependency between `Round` and `Step` classes